ci/x86_64-windows-debug.ps1

```diff
@@ -1,13 +1,21 @@
-$TARGET = "$($Env:ARCH)-windows-gnu"
+$ARCHITECTURE = switch ([Environment]::Is64BitOperatingSystem) {
+    $true { "x86_64"; break }
+    $false { "x86"; break }
+}
+
+$TARGET = "$ARCHITECTURE-windows-gnu"
```

Comment:
This script is x86_64-specific so this can be simplified, no branching logic needed:

```suggestion
$TARGET = "x86_64-windows-gnu"
```

Reply:
How about ARM64 for Windows 11? see https://www.microsoft.com/en-us/software-download/windowsinsiderpreviewARM64